Fix termhandler() signature to match __sighandler_t.

--- a/dwmblocks.c
+++ b/dwmblocks.c
@@ -34,7 +34,7 @@
 void sighandler(int signum);
 int getstatus(char *str, char *last);
 void statusloop();
-void termhandler();
+void termhandler(int signum);
 void pstdout();
 #ifndef NO_X
 void setroot();
@@ -185,7 +185,7 @@
 	writestatus();
 }

-void termhandler()
+void termhandler(int signum)
 {
 	statusContinue = 0;
 }
